src/components/Restorans/Nav/Nav.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useNavigate } from 'react-router';
+import RestoranIcon from '../../../images/Vector.png';
+import Notepad from '../../../images/notepad.png';
+import Star from '../../../images/star.png';
+import Picture from '../../../images/picture.png';
+
+const Nav = () => {
+  const navigate = useNavigate();
+  const navigateToMeni = () => {
+    navigate('/meni');
+  };
+  const navigateToRestoran = () => {
+    navigate('/topfood');
+  };
+  const navigateToGalery = () => {
+    navigate('galery');
```

Comment:
You are missing `/`, cause when you click on gallery button you should navigate to `http//:localhost:3000/gallery`, instead of the that it will navigate to `http//:localhost:3000/impressions/gallery` which doesn’t exist. Also please replace typo from `galery` to `gallery`

Reply:
Add / to navigate('/galery');